rego/rego.go

```diff
@@ -1837,6 +1837,7 @@ func (r *Rego) loadFiles(ctx context.Context, txn storage.Transaction, m metrics
 	defer m.Timer(metrics.RegoLoadFiles).Stop()
 
 	result, err := loader.NewFileLoader().
+		WithCapabilities(r.capabilities).
```

Comment:
Previously, the parser didn't have access to the capabilities file passed in via e.g. `opa eval --capabilities`. So effectively, there was no way of turning off individual future keywords.

I'm pondering breaking this out into its own bug report and PR 🤔 .

Reply:
A new PR or a separate commit should work.

Reply:
https://github.com/open-policy-agent/opa/issues/6379